### DISCUSSION TITLE
How to add a new dashboard tab

### DISCUSSION THREAD

**Discussion by PercyP** [YetiForceCompany/YetiForceCRM#15243] [How to] [answered]
Hi,

I am only just getting back in to Yetiforce. Can someone tell me how we create new dashboard tabs? I am sure there used to be an option for this but cannot see it any more. 

Any help would be greatly appreciated.

**Comment by rskrzypczak** [comment, marked as answer]
![image](https://user-images.githubusercontent.com/10221999/135640379-d1bfe3d2-7ff0-4f22-8748-522e112a2eea.png)

**Comment by PercyP** [comment]
Thank you. 

If I recall correctly, it used to be on the front end.
